.github/workflows/benchmark_trigger.yml

```diff
@@ -0,0 +1,110 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Benchmark Trigger
+
+# Trigger benchmark CI workflows when benchmark PR labels are added.
+# When a benchmark label is added, this cancels and then re-runs the CI
+# workflow. When the workflow is re-run, it will pick up the latest PR
+# description and labels. See
+# https://github.com/openxla/iree/issues/10042#issuecomment-1449250094 for
+# more background.
+#
+# This workflow is running on the main branch.
+
+on:
+  # Run when a label is added to a PR that targets the main branch.
+  pull_request_target:
+    types:
+      - labeled
+    branches:
+      - main
```

Comment:
"This workflow is running on the main branch." comment seems redundant with
```yaml
on:
...
   branches:
      - main
```

Remove the comment? Or should the workflow run for even PRs that use other base branches? (We do that sometimes with stacks of changes)

Reply:
Removed the constraint for the main branch. Updated the comment